draft-ietf-acme-acme.md

```diff
@@ -1944,11 +1932,13 @@ for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
 ".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
```

Comment:
Oh whoops, yup got mixed up about which part of the diff I was looking at. This should also be covered by the `validation servers` reference in the new top-level `Character Encoding` section.